CONTRIBUTING.rst

```diff
@@ -386,6 +386,33 @@ General guidelines for making a good gallery plot:
 * Describe the feature that you're showcasing and link to other relevant parts of the
   documentation.
 
+Adding References
+-----------------
+
+If you are contributing a new algorithm (or an improvement to a current algorithm),
+a reference paper or resource should also be provided in the function docstring.
+For references to published papers, we try to follow the `Chicago Citation Style <https://en.wikipedia.org/wiki/The_Chicago_Manual_of_Style>`__. The quickest way of generating citation in this style is
```

Comment:
```suggestion
For references to published papers, we try to follow the
`Chicago Citation Style <https://en.wikipedia.org/wiki/The_Chicago_Manual_of_Style>`__.
The quickest way of generating citation in this style is
```

Reply:
Just a silly line-breaking suggestion.

Also, it might be good to link directly to the section of the wiki article relating to citation style: https://en.wikipedia.org/wiki/The_Chicago_Manual_of_Style#Citation_styles